google/cloud/dataproc_spark_connect/session.py

```diff
@@ -280,23 +280,23 @@ def create_session_pbar():
                                 f"Exception while writing active session to file {file_path}, {e}"
                             )
                 except (InvalidArgument, PermissionDenied) as e:
-                    stop_create_session_pbar = True
+                    stop_create_session_pbar_event.set()
                     if create_session_pbar_thread.is_alive():
                         create_session_pbar_thread.join()
                     DataprocSparkSession._active_s8s_session_id = None
                     raise DataprocSparkConnectException(
                         f"Error while creating Dataproc Session: {e.message}"
                     )
                 except Exception as e:
-                    stop_create_session_pbar = True
+                    stop_create_session_pbar_event.set()
```

Comment:
This is redundant given the same call in the `finally` block.

Reply:
This is needed because we want for pbar thread to finish before we start printing errors.